homeassistant/components/thermostat/ecobee.py

```diff
@@ -26,10 +38,37 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     _LOGGER.info(
         "Loading ecobee thermostat component with hold_temp set to %s",
         hold_temp)
-    add_devices(Thermostat(data, index, hold_temp)
-                for index in range(len(data.ecobee.thermostats)))
+    devices = [Thermostat(data, index, hold_temp)
+               for index in range(len(data.ecobee.thermostats))]
+    add_devices(devices)
+
+    def fan_min_on_time_set_service(service):
+        """Set temperature on the target thermostats."""
```

Comment:
Update the docstring. Also consider changing the  name of the serivce function to `set_fan_min_on_time_service`.